snippets/Google.Pubsub.V1.Snippets/PublisherClientSnippets.cs

```diff
@@ -19,62 +19,65 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-public class PublisherClientSnippets
+namespace Google.Pubsub.V1.Snippets
 {
-    public void ListTopics()
+    public class PublisherClientSnippets
     {
-        // Snippet: ListTopics
-        PublisherClient client = PublisherClient.Create();
-
-        // Alternative: use a known project resource name:
-        // projects/{PROJECT_ID}
-        string projectName = PublisherClient.GetProjectName("PROJECT_ID");
-        foreach (Topic topic in client.ListTopicsPageStream(projectName).Flatten())
+        public void ListTopics()
         {
-            Console.WriteLine(topic.Name);
-        }
-        // End snippet
-    }
+            // Snippet: ListTopics
+            PublisherClient client = PublisherClient.Create();
 
-    public async Task ListTopicsAsync()
-    {
-        // Snippet: ListTopicsAsync
-        PublisherClient client = PublisherClient.Create();
+            // Alternative: use a known project resource name:
+            // projects/{PROJECT_ID}
```

Comment:
Maybe. I'm still not entirely happy about any of how we demonstrate resource names. I really need to write that doc... will put things in quotes for now, squash, submit and rebuild the docs.